keel-core/src/main/kotlin/com/netflix/spinnaker/keel/constraints/ManualJudgementConstraintEvaluator.kt

```diff
@@ -18,9 +20,10 @@ class ManualJudgementConstraintEvaluator(
   repository: ConstraintRepository,
   private val clock: Clock,
   override val eventPublisher: EventPublisher
-) : StatefulConstraintEvaluator<ManualJudgementConstraint>(repository) {
+) : StatefulConstraintEvaluator<ManualJudgementConstraint, DefaultConstraintAttributes>(repository) {
```

Comment:
So for every stateful constraint, we would have to pass a `constraintAttribute` object, or use this default object?

Reply:
Ya, every evaluator needs to tell us what type the attributes will be. I created this default one because not every stateful constraint needs attributes